packages/contentful--app-scripts/package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=14",
     "npm": ">=6"
   },
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
```

Comment:
This often indicates that we compile to much and the compiled test files end up in our `lib` folder. I think we can solve that by being explicit about what to compile in the tsconfig (using `files` or `includes`)

Reply:
Wdym? Previously it was a `lib` folder with all js files inside, now I moved all to `src` and output is in `lib`. We excluding test files here: https://github.com/contentful/create-contentful-app/pull/1634/files#diff-62dd7532f57cf9b72c5d182426154c8422775ea59455ea5178d2a1a4b39130c9R3

Reply:
The folder structure now looks like this

![image](https://github.com/contentful/create-contentful-app/assets/4947671/601333a3-9f13-417c-8dad-ce22ee0ab6e0)

But we normally want it to be like this

![image](https://github.com/contentful/create-contentful-app/assets/4947671/d2074843-4177-4a29-9e5a-42a42355e2da)

Probably can easily be fixed by moving the `utils` folder into the `src` folder.

Reply:
Oh nice, I didn't know how to solve that. Thanks!